classes/shortcodes.php

```diff
@@ -197,13 +228,18 @@ function do_shortcode( $args ) {
 			}
 
 			// If the requested data is a taxonomy
-			$sm_taxes = $simple_map->get_sm_taxonomies( 'array', '', true, 'objects' ) ? $simple_map->get_sm_taxonomies( 'array', '', true, 'objects' ) : array();
+			$sm_taxes = $simple_map->get_sm_taxonomies( 'array', '', true,
+				'objects' ) ? $simple_map->get_sm_taxonomies( 'array', '', true,
+				'objects' ) : array();
 
 			foreach ( $sm_taxes as $taxonomy ) {
 				if ( $taxonomy->name == $atts['data'] ) {
 					// Forward compatible format types
-					$tax_format = ( in_array( $atts['format'], array( 'csv' ) ) ) ? $atts['format'] : 'csv';
-					if ( $terms = wp_get_object_terms( $post->ID, $taxonomy->query_var ) ) {
+					$tax_format = ( in_array( $atts['format'],
+						array( 'csv' ) ) ) ? $atts['format'] : 'csv';
+					if ( $terms = wp_get_object_terms( $post->ID,
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Expected 0 spaces after opening bracket; 1 found
* Opening parenthesis of a multi-line function call must be the last content on the line

<!-- Comment by [SideCI](https://sideci.com) -->